packages/editor/src/components/post-publish-panel/index.js

```diff
@@ -97,6 +97,13 @@ class PostPublishPanel extends Component {
 						</PostPublishPanelPostpublish>
 					) }
 				</div>
+				<div className="editor-post-publish-panel__footer">
+					<CheckboxControl
+						label={ __( 'Always show pre-publish checks.' ) }
```

Comment:
Putting the word "this" inside this text might help contextualise this more.

> Always show these pre-publish checks.

might help users understand it more. If it doesn't look too cramped I say go with that, but if it's already a tight space feel free to leave it as-is.